python/ray/train/examples/jax_mnist_example.py

```diff
@@ -0,0 +1,206 @@
+import ray.train as train
+from ray.train.trainer import Trainer
+from ray.train.callbacks import JsonLoggerCallback
+
+from typing import Dict
```

Comment:
could we sort those with `isort`, and keep all ray imports in a second section?

Reply:
sorted, thanks